### ISSUE TITLE
Large output files

### ISSUE THREAD

**Issue by tsnouidui** [ibpsa/modelica-ibpsa#672]
@mlauster @mwetter All .mos scripts in https://github.com/iea-annex60/modelica-annex60/tree/master/Annex60/Resources/Scripts/Dymola/ThermalZones/ReducedOrder/Validation/VDI6007 have the `outputInterval` set to 60 seconds which leads to more than 8000 output points. Could this be reduced to about 500 points (or less) to have smaller results file which will be needed for the refactored version of `BuildingsPy`?

**Comment by mlauster** [comment]
@tsnouidui, are 1440 points sufficient? The tests are defined within the guideline that describes the model and compare hourly values for three days out of a 60 day period (day 1,10 and 60). The graphical output gets meaningless if we have an output step larger than one hour and we need all 60 days (thus 1440 steps).

**Comment by tsnouidui** [comment]
@mlauster 1440 is still large but if that is the minimum for the test we will have to go with it? @mwetter Any comment?

**Comment by mwetter** [comment]
@mlauster Can you test these 3 days * 24 hours = 72 steps within the Modelica code by checking whether you are in the right time window, and then run a test on the result versus a table that you store in the `*.mo` code? I think using the reference results for testing against some tabulated value will lead to too high a complexity as results may get interpolated or extrapolated in Buildingspy.

**Comment by carlesRT** [comment]
> have the outputInterval set to 60 seconds which leads to more than 8000 output points. Could this be reduced to about 500 points (or less) to have smaller results file [...]

Would be an option to filter the result file? keeping just a subset with the needed variables/parameters? 
using `alist.exe` or `DataFiles.writeCSVmatrix`?

**Comment by mwetter** [comment]
@carlesRT We already keep only a subset of variables. Best would be in the modelica code, use something like
```
if t0 <= time < t1 then
  // check results versus tabulated values
end when
```

**Comment by mlauster** [comment]
The current implementation of the test cases serve two purposes. 
The first one is the validation according to the given guideline. This is done in three periods (each for one day) and against table data stored in the .mo file as proposed by @mwetter. If the validation fails, the model throws an error and ends with `false`.
So far, this validation does not interfere with the second purpose, regression tests to track changes in the result files using our established workflow. Changing the output step to e.g. daily only affects the second purpose, but has some side effects. In addition, we typically have a time event per hour, so we need to disable "store variables at events":
- We would only recognize "long-term" effects that get visible in the daily time step. This shouldn't be too problematic as all major changes should force the validation to fail and the simulation to stop before termination (what would be reported by BuildingsPy, am I correct?). Simulate only the first day to catch short-term effects would not be a good idea as then we wouldn't get the full validation check.
- The graphical output when running the script in Dymola that includes comparisons to the validation results would become meaningless. Currently it looks like this:

![unbenanntes bild](https://cloud.githubusercontent.com/assets/6892690/23481624/5ddd19fc-fecc-11e6-9508-a4c260b667d4.png)

With a daily time step and no events:

![unbenanntes bild1](https://cloud.githubusercontent.com/assets/6892690/23481635/69e3298a-fecc-11e6-97a8-d1c93883e664.png)

So to sum up: If the script is not thought to be executed to get a graphical impression of the validation, then we could go down to an arbitrary time step, e.g. 10368 seconds (roughly three hours, leads to 500 points).  What do you think?

**Comment by mwetter** [comment]
@mlauster : I will check whether we can add a flag to the model that causes the time series to be not stored in the reference results (and update BuildingsPy accordingly). This should work as these models throw an assertion if they produce wrong results.

**Comment by mwetter** [comment]
Closed as the verification now works on JModelica.
